app/services/room_parser_service.py

```
@@ -174,8 +174,32 @@ def _map_day_type(keyword: Optional[str]) -> str:
+        "평일 낮": "평일 낮",
+        "평일낮": "평일 낮",
+        "평일 오전": "평일 오전",
+        "평일 야간": "평일 야간",
+        "평일": "평일",
+        "주말": "주말",
+        "공휴일": "공휴일",
+        "주말/공휴일": "주말/공휴일",
+        "심야": "심야",
+        "야간": "야간",
+        "주간": "주간",
+    }
```

Ruff (0.15.4)

[warning] Mutable default value for class attribute

(RUF012)